test/functional/inspec_check_test.rb

```diff
@@ -88,6 +88,15 @@
       out.exit_status.must_equal 1
       out.stdout.must_include 'inspec.yml and inspec.lock are out-of-sync. Please re-vendor with `inspec vendor`.'
       out.stdout.must_include 'Cannot find linux-baseline in lockfile. Please re-vendor with `inspec vendor`.'
-    end    
+    end
+  end
+
+  describe 'inspec check with invalid `include_controls` reference' do
+    it 'raises an error matching /Cannot load \'invalid_name\'/' do
+      invalid_profile = File.join(profile_path, 'invalid-include-controls')
+      out = inspec('check ' + invalid_profile)
+      out.exit_status.must_equal 1
+      out.stderr.must_match /Cannot load 'invalid_name'/
```

Comment:
I'd like to check for the string 'is not listed as a dependency' in the error message, since that is main concern of this change.

Reply:
I can add another line 👍